Wikipedia/Code/ArticleViewController.swift

```diff
@@ -1055,7 +1063,55 @@ private extension ArticleViewController {
         setupWebView()
         setupMessagingController()
     }
-    
+
+    private var overflowMenu: UIMenu {
+        let learnMore = UIAction(title: CommonStrings.learnMoreTitle(), image: UIImage(systemName: "info.circle"), handler: { [weak self] _ in
+            self?.goToFAQ()
+        })
+        
+        let tutorial = UIAction(title: CommonStrings.tutorialTitle, image: UIImage(systemName: "lightbulb.min"), handler: { [weak self] _ in
+            self?.showTutorial()
+        })
+
+        let reportIssues = UIAction(title: CommonStrings.problemWithFeatureTitle, image: UIImage(systemName: "flag"), handler: { [weak self] _ in
```

Comment:
Same for the images here - they could be WMFSFSymbolIcon. You can add a new SFSymbol if we don't have it already